press/press/doctype/server_plan/server_plan.json

```diff
@@ -0,0 +1,141 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "autoname": "prompt",
+ "creation": "2024-02-05 22:21:47.560972",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "enabled",
+  "title",
+  "pricing_section",
+  "price_inr",
+  "column_break_sjmg",
+  "price_usd",
+  "section_break_nifk",
+  "server_type",
+  "cluster",
+  "instance_type",
+  "column_break_ypkt",
+  "vcpu",
+  "memory",
+  "disk",
+  "allowed_roles_section",
+  "table_heyh"
+ ],
+ "fields": [
+  {
+   "default": "0",
+   "fieldname": "enabled",
+   "fieldtype": "Check",
+   "label": "Enabled"
+  },
+  {
+   "fieldname": "server_type",
+   "fieldtype": "Select",
+   "label": "Server Type",
+   "options": "Server\nDatabase Server"
```

Comment:
App Server instead of Server?

Reply:
I wanted this to be same as the doctype names so it's easier to filter at other places. Perhaps we can rename Server doctype to App Server later.